tests/testthat/test-adverb-auto-browse.R

```diff
@@ -1,4 +1,4 @@
 test_that("auto_browse() not intended for primitive functions", {
   expect_snapshot(auto_browse(log)(NULL), error = TRUE)
-  expect_error(auto_browse(identity)(NULL), NA)
+  expect_snapshot(auto_browse(identity)(NULL))
```

Comment:
Here we are not testing an error message but the absence of error so this is clearer about the intention:

```suggestion
  expect_no_error(auto_browse(identity)(NULL))
```